README.md

```diff
@@ -1,117 +1,69 @@
-* Delete your branch after merging it. This keeps the repo clean and faster to sync.
-* Use an exception if this is an error. Only use `assert` for debugging or proof-checking purpose. This is because exception messages usually contain more information.
-* Use modern python features and styles that increases code quality.
-  * Use f-string instead of `.format()` for short expressions to increase readability.
-  * Use `class MyClass:` instead of `class MyClass(object):`. The later one was a workaround for python2.x.
-  * Use `abc` module for abstract classes to ensure all abstract methods are implemented.
-  * Use python typing. But you should not import external objects just for typing. Instead, import typing-only external objects under `if typing.TYPE_CHECKING:`.
-
-### Testing
-To run smoke tests:
-```
-bash tests/run_smoke_tests.sh
-
-# Run one of the smoke tests
-bash tests/run_smoke_tests.sh test_minimal
+## Getting Started
+You can find our documentation [here](https://skypilot.readthedocs.io/en/latest/).
+- [Installation](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html)
+- [Quickstart](https://skypilot.readthedocs.io/en/latest/getting-started/quickstart.html)
+- [CLI reference](https://skypilot.readthedocs.io/en/latest/reference/cli.html)
+
+## Example SkyPilot Task
+
+Tasks in SkyPilot are specified as a YAML file containing the resource requirements, data to be synced, setup commands and the task commands. Here is an example.
+
+```yaml
+# my-task.yaml
+resources:
+  # 1x NVIDIA V100 GPU
+  accelerators: V100:1
+
+# Number of VMs to launch in the cluster
+num_nodes: 1
+
+# Working directory (optional) containing the project codebase.
+# Its contents are synced to ~/sky_workdir/ on the cluster.
+workdir: .
+
+# Commands to be run before executing the job
+# Typical use: pip install -r requirements.txt, git clone, etc.
+setup: |
+  echo "Running setup."
+
+# Commands to run as a job
+# Typical use: make use of resources, such as running training.
+run: |
+  echo "Hello, SkyPilot!"
+  conda env list
 ```
 
-For profiling code, use:
-```
-pip install tuna # Tuna for viz
-python3 -m cProfile -o sky.prof -m sky.cli status # Or some other command
-tuna sky.prof
+This task can be launched on the cloud with the `sky launch` command.
+```bash
+$ sky launch my-task.yaml
 ```
+SkyPilot will perform multiple functions for you:
+1. Find the lowest priced VM instance type across different clouds
+2. Provision the VM
+3. Copy the local contents of `workdir` to the VM
+4. Run the task's `setup` commands to prepare the VM for running the task 
+5. Run the task's `run` commands
+
+<!---- TODO(romilb): Example GIF goes here ---->
+Please refer to the [quick start](https://skypilot.readthedocs.io/en/latest/getting-started/quickstart.html) and [documentation](https://skypilot.readthedocs.io/en/latest/) for more on how to use SkyPilot.
+
+## Contributing
```

Comment:
I think before this section, we should add a `Issues and feature requests` section -- briefly mentioning we welcome these in GitHub issues and discussions (1 sentence).  Wdyt?

Reply:
Done!